app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -4,14 +4,15 @@ module Api
   module V1
     module Users
       class OmniauthCallbacksController < Devise::OmniauthCallbacksController
-        rescue_from Exception, with: :render_error
-
         respond_to :json
 
         def google_oauth2
           user = User.from_omniauth(auth)
```

Comment:
A lin ebreak would nice to separate the variable definition and conditional block